apps/public-docsite-v9/src/Concepts/Poster/Legend.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+
+import { makeStyles, shorthands, tokens } from '@fluentui/react-components';
+import { ComponentIcon, ConstantIcon, HookIcon, MethodIcon, TypeIcon } from './CodeItemIcons';
+
+const useStyles = makeStyles({
+  root: {
```

Comment:
Feature request: It would be nice if the legend were fixed and floated over the top of the poster so that it would always be visible. Right now it's hidden off on the bottom left of the screen.